packages/kokkos/cmake/kokkos_tpls.cmake

```
@@ -2,22 +2,44 @@ KOKKOS_CFG_DEPENDS(TPLS OPTIONS)
 KOKKOS_CFG_DEPENDS(TPLS DEVICES)
 
 FUNCTION(KOKKOS_TPL_OPTION PKG DEFAULT)
+  CMAKE_PARSE_ARGUMENTS(PARSED
+    ""
+    "TRIBITS"
+    ""
+    ${ARGN})
+
+  IF (PARSED_TRIBITS)
+    #this is also a TPL option you can activate with Tribits
+    #Tribits brings its own default
+    SET(DEFAULT TPL_ENABLE_${PARSED_TRIBTS})
```

@jjwilke I think my typo fix here got wiped out
```suggestion
    SET(DEFAULT TPL_ENABLE_${PARSED_TRIBITS})
```